MovieQuiz/Presentation/AlertPresenter.swift

```diff
@@ -0,0 +1,22 @@
+import UIKit
+
+class AlertPresenter: AlertPresenterDelegate {
+    
+    weak var delegate: AlertPresenterProtocol?
+    init(delegate: AlertPresenterProtocol) {
```

Comment:
Логические блоки принято разделять одной пустой строкой
```suggestion
    weak var delegate: AlertPresenterProtocol?
    
    init(delegate: AlertPresenterProtocol) {
```